### PR TITLE
Allow General Calls on I2C bus (aka broadcast)

### PR DESCRIPTION
**Pull request by agdl**
This allow to call a Wire.begin(address, true) to enable general call (broadcast messages on address 0x00) feature on SAMD21 based devices. All the legacy calls to Wire.begin(address) are ok since the extra parameter is false by default

### CODE REVIEW

File: cores/arduino/SERCOM.cpp [hunk 1]
```diff
@@ -400,17 +400,20 @@ void SERCOM::disableWIRE()
   }
 }
 
-void SERCOM::initSlaveWIRE( uint8_t ucAddress )
+void SERCOM::initSlaveWIRE( uint8_t ucAddress, bool broadcast )
```
Comment (facchinm) [inline cores/arduino/SERCOM.cpp]: Should we call the parameter `listen_on_broadcast` or something similar? @sandeepmistry

Comment (sandeepmistry) [reply cores/arduino/SERCOM.cpp]: Maybe `enableGeneralCall`?

**Comment by agdl** [comment]
@lupalby I think you will like this too ;) https://github.com/agdl/WireUpdate

**Comment by lupalby** [comment]
@agdl That's a cool feature! I will look into it

**Pull request merged**